src/pages/overlay/components/ambassadorList/AmbassadorList.tsx

```diff
@@ -80,18 +76,17 @@ export default function AmbassadorList(props: AmbassadorListProps) {
               key={key}
               ambassadorKey={key}
               ambassador={ambassador}
-              getCard={() => {
+              onClick={() => {
                 setActiveAmbassador(prev => prev === key ? undefined : key)
               }}
-              ClassName={`${styles.ambassadorButton} ${activeAmbassador === key ? styles.ambassadorButtonClicked : undefined}`}
-              Id={key}
+              className={`${styles.ambassadorButton} ${activeAmbassador === key ? styles.highlighted : undefined}`}
```

Comment:
`undefined` gets stringified here (not a new problem)

```suggestion
              className={`${styles.ambassadorButton} ${activeAmbassador === key ? styles.highlighted : ''}`}
```

Reply:
Implemented the pattern we use in the site for class filtering